src/lib.rs

```diff
@@ -196,20 +198,25 @@ impl Renderer {
     /// The texture is assumed to be of the specified dimensions and have been created with
     /// the [wgpu::TextureFormat::Rgba8Unorm] format and the [wgpu::TextureUsages::STORAGE_BINDING]
     /// flag set.
+    ///
+    /// The return value is the value of the BumpAllocators in this rendering, which is currently used
```

Comment:
Returning BumpAllocators is exposing too much low-level detail, though obviously useful for development for the moment. I'd add a comment suggesting that the exact type may change, for example to more of an opaque stats structure.